### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by OlawaleJoseph**
## What does this PR do?
This Pr implements the core functionality of this calculator app

### Implementations
- Created a module to handed basic arithmetic operations.
- Created a module to handle the core logic of the application
- Implemented result chaining

**NOTE**
I implemented some features of mine which needed me to create some extra variables and methods, which does not affect the project specifications. Some of the features include result chaining and memorization of previous calculation.

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+  if (btn === '%') {
+    if (prev && !operator) {
+      total = (prev / 100).toString();
+      prev = '0';
+    }
+    if (next) {
+      next = (next / 100).toString();
+    }
+  }
```
Comment (MuriloRoque) [inline src/logic/calculate.js]: - [x] In this case, you should use an operation from the `operate` function. Create a conditional there for this case and call the `modifier` function from the `Big.js` library in order to convert the number to a percentage 😉

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested 🔴 

### Hello @OlawaleJoseph 👋 

### Great job in this milestone, it's almost perfect, there's only one minor issue. I made a comment in your code, good luck! 😄 

Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @OlawaleJoseph :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

File: src/logic/calculate.js (hunk 1 above)
Comment (OlawaleJoseph) [reply src/logic/calculate.js]: Fixed. Thanks for the review

**Pull request merged**